### ISSUE TITLE
Snackbar Inside Charges Dialog 

### ISSUE THREAD

**Issue by aj019** [openMF/android-client#654]
**Summary:** 

While creating new charges if there is an error in the Charges Dialog then a snackbar is shown inside the Dialog and the content of the Dialog go invisible. It would be better to show a Toast in this situation.

**Steps to reproduce:** 

Open the app
Select Clients from the Navbar
Select Client 
From the toolbar menu select Charges
From the toolbar menu click on '+' icon
Now click on save without entering the amount

**Expected behavior:** 

The snackbar shouldn't be inside the Dialog but it should be on the bottom of the screen or a Toast should appear.

**Observed behavior:** 

A snackbar appears inside the dialog and the fields go invisible

**Device and Android version:** 

Nexus 5 API 23

**Screenshots:** 

![ezgif com-video-to-gif](https://cloud.githubusercontent.com/assets/7893859/24584713/c6848bf4-1793-11e7-8a76-adf1ced8e8bc.gif)

**Comment by aj019** [comment]
I am working on this issue.

**Comment by mayank-kgp** [comment]
@therajanmaurya @tarun0 @puneetkohli521 I don't think that there is any problem if snack bar is shown inside the dialogue.
What do you say?

**Comment by sats98** [comment]
@therajanmaurya  i would like to on this..
